docs/changelog.txt

```diff
@@ -85,9 +85,16 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - ``Buildings::findCivzonesAt()``: lookups now complete in constant time instead of linearly scanning through all civzones in the game
 - ``Job::remove_postings()``: use the job cancel vmethod graciously provided by The Toady One in place of a synthetic method derived from reverse engineering
 
+## API
```

Comment:
Since a release has passed, could you move the changelog lines up to the new "Future" section?